roles/hammerdb/templates/db_creation.yml

```diff
@@ -1,20 +1,35 @@
 apiVersion: batch/v1
 kind: "job"
 metadata:
-  name: "{{ ansible_operator_meta.name }}-creator-{{ trunc_uuid }}"
+  name: "{{ meta.name }}-creator-{{ trunc_uuid }}"
```

Comment:
I believe `ansible_operator_meta` should be correct, did something change?

Reply:
You right, fix it.